qase v5.x - qaseio/commons/interceptor.py fails with AttributeError: 'NoneType' object has no attribute 'data'

Test example:
```python
import requests
from qaseio.pytest import qase

@qase.id(2831)
@qase.title("My test")
def test_example():
    requests.get('https://google.com')
    assert True
```
Config
```json
{
	"mode": "testops",
	"testops": {
		"api": {
			"token": "my_token"
		},
		"project": "DEMO"
	},
	"framework": {
		"pytest": {
			"capture": {
				"logs": true,
				"http": true
			}
		}
	}
}

```
Exception
```
test_examples.py::test_example FAILED                                    [100%]
test_examples.py:5 (test_example)
@qase.id(2831)
    @qase.title("My test")
    def test_example():
>       requests.get('https://google.com')

test_examples.py:9: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
venv/lib/python3.11/site-packages/requests/api.py:73: in get
    return request("get", url, params=params, **kwargs)
venv/lib/python3.11/site-packages/requests/api.py:59: in request
    return session.request(method=method, url=url, **kwargs)
venv/lib/python3.11/site-packages/requests/sessions.py:589: in request
    resp = self.send(prep, **send_kwargs)
venv/lib/python3.11/site-packages/qaseio/commons/interceptor.py:47: in wrapper
    InterceptorSingleton.get_instance()._log_post_response(response)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

response = <Response [200]>, url = None, args = (), kwargs = {}, status = 200
interceptor = <qaseio.commons.interceptor.Interceptor object at 0x10648be50>

    @staticmethod
    def _log_post_response(response, url=None, *args, **kwargs):
        status = response.status if hasattr(response, 'status') else response.status_code
        interceptor = InterceptorSingleton.get_instance()
>       interceptor.step.data.add_response(
            status_code = status,
            response_body = str(response.data if hasattr(response, 'data') else response.content) if interceptor.track_on_fail and status >= 400 else None,
            response_headers = response.headers if interceptor.track_on_fail and status >= 400 else None,
        )
E       AttributeError: 'NoneType' object has no attribute 'data'

venv/lib/python3.11/site-packages/qaseio/commons/interceptor.py:83: AttributeError
```

--- a/qase-python-commons/pyproject.toml
+++ b/qase-python-commons/pyproject.toml
@@ -4,7 +4,7 @@ build-backend = "setuptools.build_meta"
 
 [project]
 name = "qase-python-commons"
-version = "2.0.7"
+version = "2.0.8"
 description = "A library for Qase TestOps and Qase Report"
 readme = "README.md"
 authors = [{name = "Qase Team", email = "support@qase.io"}]

--- a/qase-python-commons/src/qaseio/commons/interceptor.py
+++ b/qase-python-commons/src/qaseio/commons/interceptor.py
@@ -80,6 +80,8 @@ class Interceptor:
     def _log_post_response(response, url=None, *args, **kwargs):
         status = response.status if hasattr(response, 'status') else response.status_code
         interceptor = InterceptorSingleton.get_instance()
+        if interceptor.step is None:
+            return
         interceptor.step.data.add_response(
             status_code = status,
             response_body = str(response.data if hasattr(response, 'data') else response.content) if interceptor.track_on_fail and status >= 400 else None,

